### ISSUE TITLE
Hhe problem of feedback

### ISSUE THREAD

**Issue by BillTaozi** [AlanFinotty1995/AAF_DCH_RealtekModded#35]
Nahimic 3 1.4.2.0
 Sonic Studio 3 3.16.12.0
Can’t these two be installed simultaneously?

**Comment by AlanFinotty1995** [comment]
No, because they both use the same libraries and audio processing services (APO).
